README.md

```diff
@@ -1,2 +1,74 @@
-# Mobile-portofolio
- This is the setup and 1st modification of the mobile portfolio project which I will be working on.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
```

Comment:
- [ ] Kindly add your project name `# Portfolio` instead of `# Project Name` to make your readme file descriptive 👍